File: Problems/hacker_rank/Arrays_and_Sorting/20141226_Intro_to_Tutorial_Challenges/Solution.scala
object Solution {
  def main(args: Array[String]) {
    val inps = io.Source.stdin.getLines.toArray
    val T = inps(0).toInt
    val n = inps(1).toInt
    val arr = inps(2).split(" ").map((s: String) => s.toInt)
    for ( i <- 0 until arr.length ) {
      if ( T == arr(i) )  {
        println(i)
        return
      }
    }
  }
}

File: Problems/hacker_rank/Arrays_and_Sorting/20141226_Intro_to_Tutorial_Challenges/Solution.java
import java.util.Scanner;

public class Solution {
  public static void main(final String[] args)  {
    Scanner sc = new Scanner(System.in);
    final int V = Integer.parseInt(sc.nextLine());
    final int n = Integer.parseInt(sc.nextLine());
    final String[] arr = sc.nextLine().split(" ");
    for ( int i = 0; i < arr.length; ++i )  {
      if ( V == Integer.parseInt(arr[i]) )  {
        System.out.println(i);
      }
    }
  }
}
